### ISSUE TITLE
imagnet

### ISSUE THREAD

**Issue by xiaoxutongxu** [wangkiw/ICLR23-MEMO#3]
Thank you for your excellent work. I am currently reproducing your experiments on ImageNet100 using the default settings provided in your code. However, when running the 50-5 setting, I noticed a significant performance drop starting from the second task, and the results differ considerably from those reported in your paper.

To better understand this discrepancy, could you kindly share the detailed parameter settings you used for the ImageNet100 experiments, especially for the 50-5 configuration? Any additional information such as training epochs, batch size, memory size, learning rate schedule, or data preprocessing details would be greatly appreciated.

Thank you very much for your time and help.

Here are the parameters I used for the 50-5 setting on ImageNet100:

-model memo -init 50 -incre 5 -ms 3312 -net memo_resnet18 -d 3 -p fair --train_base -d 9
